__tests__/index.test.js

```diff
@@ -68,3 +68,40 @@ describe("/api/reviews", () => {
       });
   });
 });
+
+describe("/api/reviews/:review_id", () => {
+  test("GET - 200: Responds with an an object corresponding to the queried review_id", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.selectedReview).toMatchObject({
+          review_id: expect.any(Number),
```

Comment:
This test should be asserting that the review with id 1 is returned, not just any id. You also know what this should look like from the test database, so feel free to assert against the values you know should be returned.